tests/ppl/explain.yaml

```diff
@@ -0,0 +1,22 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test how a query is executed against OpenSearch.
+
+prologues:
+  - path: /books
+    method: PUT
+    request_body:
+      payload: {}
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Get explain of SQL Query
```

Comment:
Add a period. We should really add it to the linter :)

Reply:
Yes, I will try to finish the linter, because for me it happens so often :)

Reply:
I had a PR almost ready for some of it, https://github.com/opensearch-project/opensearch-api-specification/pull/463.